src/inspectors/VoIPInspector.tsx

```diff
@@ -0,0 +1,132 @@
+/*
+Copyright 2023 New Vector Ltd
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { MatrixClient } from "matrix-js-sdk/src/client";
+import { GroupCall } from "matrix-js-sdk/src/webrtc/groupCall";
+import { CallFeed } from "matrix-js-sdk/src/webrtc/callFeed";
+import React from "react";
+
+import styles from "./VoIPInspector.module.css";
+
+interface VoIPViewerProps {
+  client: MatrixClient;
+  groupCall: GroupCall;
+  userMediaFeeds: CallFeed[];
+  screenshareFeeds: CallFeed[];
+}
+
+export function VoIPViewer({
```

Comment:
Maybe 'Media' would be a better name for this tab? The whole app is arguably VoIP.

Reply:
Renamed it to Media